src/helpers/index.tsx

```diff
@@ -306,6 +308,53 @@ export const handleContractError = (e: any) => {
   if (EnvHelper.env.NODE_ENV !== "production") console.warn("caught error in slices; usually network related", e);
 };
 
+/**
+ * Converts a supported networkId to its hexadecimal value as a string
+ */
+export const networkIdToHexString = (id: NetworkId) => "0x" + id.toString(16);
+
+/**
+ * Determines if app is viewed within an <iframe></iframe>
+ */
+export const isIFrame = () => window.location !== window.parent.location;
```

Comment:
we have an implementation of this in `web3Context`. Can you remove the one in web3Context?

Reply:
scratch that, let's leave that as a TODO in a later pr to keep this one smaller.

Reply:
Agreed, later PR it is